pkg/drivers/powerflex_test.go

```diff
@@ -194,3 +196,126 @@ func TestModifyPowerflexCR(t *testing.T) {
 		})
 	}
 }
+
+func TestExtractZonesFromSecret(t *testing.T) {
+	emptyConfigData := ``
+	invalidConfigData := `
+- username: "admin"
+	-
+`
+	dataWithNoSystemID := `
+- username: "admin"
+  password: "password"
+  endpoint: "https://127.0.0.2"
+  skipCertificateValidation: true
+  mdm: "10.0.0.3,10.0.0.4"
+`
+	dataWithZone := `
+- username: "admin"
+  password: "password"
+  systemID: "2b11bb111111bb1b"
+  endpoint: "https://127.0.0.2"
+  skipCertificateValidation: true
+  mdm: "10.0.0.3,10.0.0.4"
+  zone:
+    label: topology.kubernetes.io/zone=US-EAST
```

Comment:
Secret should have both 'labelKey' and 'name' fields, I think-- check with @tdawe or double-check the latest proposed spec

Reply:
Yes, you're right. Thanks! confirmed with Trevor. Was confused with a different acceptance criteria:
Fixed. Also, fixed the AC.